lib/socialcast-git-extensions/cli.rb

```diff
@@ -311,6 +311,7 @@ def release
         run_cmd "git pull . #{branch}"
         run_cmd "git push origin HEAD"
         integrate_branch(base_branch, staging_branch)
+        integrate_branch(base_branch, prototype_branch) unless enforce_staging_before_release?
```

Comment:
I can't think of a way that removing the conditional would be disruptive, but no worries if you want to leave it this way 😄